docs/site/Authorization-component-authorizer.md

```diff
@@ -62,6 +62,16 @@ export class MyApplication extends BootMixin(
   constructor(options: ApplicationConfig = {}) {
     super(options);
 
+    const authoptions: AuthorizationOptions = {
+      precedence: AuthorizationDecision.DENY,
+      defaultDecision: AuthorizationDecision.DENY,
+    };
+
+    // mount authorization component
+    const binding = this.component(AuthorizationComponent);
+    // configure authorization component
+    this.configure(binding.key).to(authoptions);
+    
```

Comment:
I believe the code lint error is about the extra space in this line.

Reply:
@albjeremias, I was trying to fix the lint error in your branch, but didn't have the permission to push. Anyway, if you run `npm run lint --fix`, it should fix the problem. Thanks.